### PR TITLE
Exclude

### PR DESCRIPTION
**Pull request by saichint**
This PR is for including some properties which were excluded from certain platforms and software versions due to platform limitations and issues. All beaker tests pass.

### CODE REVIEW

File: tests/beaker_tests/lib/utilitylib.rb [hunk 1]
```diff
@@ -699,6 +709,21 @@ def supported_property_hash(tests, id, property_hash)
     # because :resource hash currently uses strings for keys
     copy.delete(prop_symbol.to_s)
   end
+  return copy if version_unsupported_properties(tests, id).empty?
+  lim = full_version.split[0].tr('(', '.').tr(')', '.').chomp('.')
+  # due to a bug in Gem::Version, we need to append a letter
+  # to the version field if the to be compared version
+  # has a letter at the end
```
Comment (mikewiebe) [inline tests/beaker_tests/lib/utilitylib.rb]: Same comment as the NU side here.  Please add an example of real versions in the comments for this.

File: tests/beaker_tests/cisco_bgp/test_bgp.rb [hunk 2]
```diff
@@ -258,6 +258,20 @@ def unsupported_properties(tests, id)
   unprops
 end
 
+def version_unsupported_properties(_tests, _id)
+  unprops = {}
+  if platform[/n7k/]
+    unprops[:disable_policy_batching_ipv4] = '8.1.1'
```
Comment (mikewiebe) [inline tests/beaker_tests/cisco_bgp/test_bgp.rb]: Does this mean that  `disable_policy_batching_ipv4` is not supported on `8.1.1` or it's only supported on `8.1.1` and later?

**Comment by mikewiebe** [comment]
@saichint One question and one comment.  Once I have the answer to the question, I may have more comments.

File: tests/beaker_tests/cisco_bgp/test_bgp.rb (hunk 2 above)
Comment (saichint) [inline tests/beaker_tests/cisco_bgp/test_bgp.rb]: It is supported on 8.1.1 and later

File: tests/beaker_tests/lib/utilitylib.rb (hunk 1 above)
Comment (saichint) [inline tests/beaker_tests/lib/utilitylib.rb]: Added

File: tests/beaker_tests/cisco_bgp/test_bgp.rb (hunk 2 above)
Comment (saichint) [inline tests/beaker_tests/cisco_bgp/test_bgp.rb]: Added comments, examples to version_unsupported_properties method in utilitylib.rb file as well

**Comment by mikewiebe** [comment]
rubocop errors
```
tests/beaker_tests/lib/utilitylib.rb:694:57: C: Trailing whitespace detected.

# Ex: If property 'ipv4_sub_option_circuit_id_string' is 
```

**Pull request merged**